Automattic/wp-calypso
# Testing: Refactor signup tests to remove rewire dependency

Related: #1515

This pull request seeks to refactor `client/signup` to remove its dependency on `rewire`, in an effort to move forward with Calypo's Babel 6 migration.

__Testing instructions:__

Ensure that Mocha tests continue to pass.

Verify that no regressions are introduced to signup.

/cc @gziolo , @umurkontaci
